src/include/homestore/btree/detail/simple_node.hpp

```diff
@@ -225,63 +225,67 @@ class SimpleNode : public VariantNode< K, V > {
     }
 
     std::string to_string_keys(bool print_friendly = false) const override {
-        std::string delimiter = print_friendly ? "\n" : "\t";
```

Comment:
You can remove this function altogether, it was not correct code and it was merged here by mistake.

Reply:
this part is now wrapped by  `#if 0 and #endif`, it can be removed when revisiting here and implement this function

Reply:
@JacksonYao287 Please wrap it with #if 0 because it is still beneficial in some cases. Do it for all to_string_keys()

Reply:
sure
@shosseinimotlagh